src/pools/D3MAaveDai.t.sol

```diff
@@ -945,11 +945,11 @@ contract D3MAaveDaiTest is DSTest {
         vat.cage();
 
         // Simulate DAI holder gets some gems from GS
-        vat.grab(ilk, address(directDepositHub), address(this), address(this), -int256(100 ether), -int256(0));
+        vat.grab(ilk, address(d3mAaveDaiPool), address(this), address(this), -int256(100 ether), -int256(0));
 
         // User can exit and get the aDAI
         directDepositHub.exit(ilk, address(this), 100 ether);
-        assertEqApprox(adai.balanceOf(address(this)), 100 ether, 1);     // Slight rounding error may occur
+        assertEqApprox(adai.balanceOf(address(this)), d3mAaveDaiPool.convertToShares(100 ether), 1);     // Slight rounding error may occur
```

Comment:
@hexonaut I'm not sure about this change. would you mind taking a look to ensure this is still the way we want aave to work.

Related change in hub: https://github.com/makerdao/dss-direct-deposit/pull/31/files#diff-c4ad6fc42729111dec6134dd5b4f5dc31016211ad92b7748de2b68179c57d5d3R370